content/en/docs/troubleshooting.md

```diff
@@ -12,91 +12,89 @@ To suggest a troubleshooting item, [create a pull request PR with your suggested
 
 ## How do I troubleshoot issues with signatures?
 
-Inspect the descriptor of a remote artifact sent for signing using  `oras manifest fetch`. For example:
+`oras manifest fetch` fetches information about an image; such as mediatype, config, layers, and annotations. By using the `--descriptor` flag, you can fetch only the descriptor of an artifact. Inspect the descriptor of a remote artifact sent for signing using `oras manifest fetch --descriptor`. For example: 
```

Comment:
`oras manifest fetch` fetches information about an artifact, image is a type of artifact, there are other types of artifacts.

Reply:
And you use `artifact` in the sentence after the first sentence, so it's better to be consistent.